release-blog/2024/2024-08-20.0.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: 2024-08-20.0
+slug: latest
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import QueryBuilder from '@site/src/components/queryBuilder';
+
+
+**Overture's `2024-08-20.0` data release is now available.** The base, buildings, divisions, and places themes are stable, which means the only changes this month are data improvements and bug fixes. The transportation theme remains in beta and we have made several significant changes to the schema in this release. The addresses theme is in alpha.
+
+The datasets are available as GeoParquet files stored on AWS and Azure. The release paths are:
+
+##### Microsoft Azure:
+```
+wasbs://release@overturemapswestus2.blob.core.windows.net/2024-08-20.0
+```
+
+##### Amazon S3:
+```
+s3://overturemaps-us-west-2/release/2024-08-20.0
+```
+
+You can access the datasets by following the process outlined [here](https://docs.overturemaps.org/getting-data/). We encourage users to report bugs and provide feedback on the data, schema, and GERS on one of our Overture GitHub repositories:
+
+- [Schema](https://github.com/OvertureMaps/schema)
+- [Data](https://github.com/OvertureMaps/data)
+- [Docs](https://github.com/OvertureMaps/docs)
+- [Python command-line tool](https://github.com/OvertureMaps/overturemaps-py)
+- [Explore site](https://github.com/OvertureMaps/io-site)
+
+
+
+### Global Entity Reference System (GERS) changelog
+
+The GERS changelog captures any changes in Overture features between this release and the previous release. The changelog is available as Parquet files &mdash; partitioned by theme, type, and change type &mdash; at the following locations on Azure and AWS:
+
+```
+wasbs://changelog@overturemapswestus2.blob.core.windows.net/2024-08-20.0
+```
+```
+s3://overturemaps-us-west-2/changelog/2024-08-20.0
+```
+
+You can find more information about [using the GERS Changelog in our documentation](https://docs.overturemaps.org/gers/changelog/). 
+
+
+
+### Breaking changes
+
+Changes were made to the transportation theme in this release to better support a variety of routing use cases. A summary of those changes, below:
```

Comment:
@jeffdefacto anything you think we should add to this section, or the theme sections below?

Reply:
Maybe add a link to the release blog where you mention it